Loop/Managers/DeviceDataManager.swift

```diff
@@ -1210,6 +1211,36 @@ extension DeviceDataManager {
                 case .cancelTemporaryOverride:
                     log.default("Canceling temporary override from remote command")
                     loopManager.settings.scheduleOverride = nil
+                case .bolusEntry(let bolusValue):
```

Comment:
We should trigger a user notification when this happens. Or is the push notification including foreground text?

Reply:
I've added notifications to indicate success and failure once the remote notification is processed. That should handle the foreground notification and allow for better troubleshooting for both foreground and background states. I've added screenshots and more discussion in Zulip https://loop.zulipchat.com/#narrow/stream/144182-development/topic/Remote.20bolus.20.2F.20remote.20carbs